python/py-buildbot-worker/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-buildbot-worker
+version             2.3.1
+revision            0
+
+maintainers         nomaintainer
+description         Buildbot Worker Daemon
+long_description    ${description}
+
+platforms           darwin
+supported_archs     noarch
+
+homepage            https://buildbot.net/
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+license             
+
+checksums           sha256  a26c68debb70f15abee307aff7b225e91a2eedca9c8d571212c391e615c36f53 \
+                    rmd160  02688cb6233c80e46038a528f5e0b938353ef325 \
+                    size    110554
+
+python.versions     36 37
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_lib-append \
+                    port:py${python.version}-future \
+                    port:py${python.version}-twisted
+
+    depends_test-append \
```

Comment:
`depends_test` is of course calling for `test.run yes`, but I assume you know that already :)